x/ccv/provider/keeper/relay.go

```diff
@@ -285,15 +292,19 @@ func (k Keeper) OnRecvSlashPacket(ctx sdk.Context, packet channeltypes.Packet, d
 func (k Keeper) ValidateSlashPacket(ctx sdk.Context, chainID string,
```

Comment:
I'm adding this here as I cannot add it to `HandleSlashPacket`.

When handling a slash packet, drop it if `providerConsAddr` is in `SlashAck(chainID)`

Reply:
Done.